doc/tutorial/introduction/index.md

```diff
@@ -0,0 +1,119 @@
+in separate browser tabs such that you can easily switch back to the tutorial.
+
+<br/>
+
+---
+
+# Read sequence files
+
+SeqAn3 provides fast and easy access to biological file formats.
+The following code example demonstrates the interface of seqan3::sequence_file_input.
+Can you imagine anything easier? After you have initialised the instance with a filename,
+you can simply query the data of interest using the get function. The available fields are
+`SEQ`, `ID` and `QUAL` to retrieve sequences, ids and qualities, respectively. The latter is empty,
+unless you read FastQ files. The appropriate file format is detected by SeqAn from your file name suffix.
+
+\snippet introduction_align.cpp sequence_input_include
+\snippet introduction_align.cpp sequence_input
+
+Here is the content of `seq.fasta`, so you can try it out!
+
+~~~
+>seq1
+ACGTGATG
+>seq2
+AGTGATACT
+~~~
+
+<br/>
+
+---
+
+# Align two sequences
+
+We have two sequences from the file above now – so let us align them.
+The pairwise sequence alignment is one of the core algorithms in SeqAn, and used by several library components
+and apps. It is strongly optimised for speed and parallel execution, while providing exact results and a
+generic interface.
+
+\snippet introduction_align.cpp alignment_include
+\snippet introduction_align.cpp alignment
+
+The algorithm returns a range of result objects – which is the reason for the loop here (in this case the range
+has length 1). Instead of passing a single pair of sequences, we could give a vector of sequence pairs to the
+algorithm, which executes all alignments in parallel and stores the results in various seqan3::align_result
+objects. The second argument to seqan3::align_pairwise is the *configuration*, which allows you to specify
+a lot of parameters for the alignment computation, for instance score functions, banded alignment and whether
+you wish to compute a traceback or not. The configurations have their own namespace seqan3::align_cfg and can
+be piped for building combinations. Check the alignment tutorial, if you want to learn more.
+
+\note
+Every directory in the SeqAn sources contains an `all.hpp` file, which includes all the functionality
+of the respective (sub-) module. Thus, if you are working with several alignment methods, you can simply
+include `seqan3/alignment/all.hpp` instead of naming each file individually.
+
+<br/>
+
+---
+
+# Indexed searching
+At the end of this introduction tutorial, we want to present you another efficient algorithm in SeqAn:
+The following code example shows you the steps for performing rapid searches in an indexed genome sequence.
+As index we use the FM index here. After creation we want to query the positions of the motiv *TAG* inside
+a given sequence.
+
+\snippet introduction_align.cpp index_search_include
+\snippet introduction_align.cpp index_search
+
+For more functions on that we refer you to the *Index Tutorial*. We hope, you have got a rough overview on
```

Comment:
```suggestion
For more information on indexing and searching we refer you to the *Index Tutorial*. We hope you got a rough overview on
```

Reply:
I'll also make a todo in my PR to change this to link to the tutorial

Reply:
Cool! 👍